### PR TITLE
FooterBar Wireup

### PR DESCRIPTION
**Pull request by gblakeman**
The bulk of this work is focused on wiring up the `FooterBar` component’s action buttons (and hiding a few buttons that will come in later work).

* Repurposed the existing `remixAll` action from 1.0 for use with the “remix” button in the footer.
* Added the `stepBack` action for triggering Figma’s undo function and reversing the focused item in the plugin UI. At the moment, this only allows for a single step back, but future work could track multiple steps.
* Added a few re-usable convenience functions to `App` – `resetState`, `setAdvanceItem` (with direction), and `setUndo` to consolidate some code and make actions less redundant.
* Both `dispatch` and `GUI` get a few new cases to manage new states/actions related to this work.
* Added our `eventsNotToTrack` convention to the analytics setup to omit a few things.
* Fixed a few bugs from earlier work.

----

### CODE REVIEW

File: manifest.example.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
   "menu": [
     {
       "name": "Show PLUGIN_NAME",
-      "command": "tools"
+      "command": "show-plugin"
```
Comment (gblakeman) [inline manifest.example.json:12]: Renamed for better analytics.

Comment (lynnerang) [reply manifest.example.json:12]: Cooool, I might poach this for Specter!

File: src/views/BottomTray.svelte [hunk 2]
```diff
@@ -3,6 +3,7 @@
   import FooterBar from './FooterBar';
   import ItemsList from './ItemsList';
 
+  export let hasHistory = false;
```
Comment (lynnerang) [inline src/views/BottomTray.svelte:6]: Could have `boolean` type here if you wanna match w/ FooterBar

**Review by lynnerang — APPROVED** [review]
Noice, LGTM!

File: src/views/BottomTray.svelte (hunk 2 above)
Comment (gblakeman) [reply src/views/BottomTray.svelte:6]: Ah, yes, good catch – thanks! Slowly been adding types to everything, but sometimes forget…

**Review by joeyyang — APPROVED** [review]
Looks good! A few naming / docs comments.

**Pull request merged**